pkg/operator/rgw/rgw.go

```diff
@@ -113,7 +113,12 @@ func (c *Cluster) createKeyring(clientset *kubernetes.Clientset, cluster *mon.Cl
 	secrets := map[string]string{
 		keyringName: keyring,
 	}
-	_, err = clientset.Secrets(c.Namespace).Create(&v1.Secret{ObjectMeta: v1.ObjectMeta{Name: appName}, StringData: secrets})
+	secret := &v1.Secret{
```

Comment:
the appName is actually different (mon vs rgw). However, I should at least define the `kubernetes.io/rook` in a single place.